src/pages/stats/[login].tsx

```diff
@@ -80,35 +151,43 @@ export default function Stats() {
       {isLoading ? (
         <div>Loading...</div>
       ) : format === "cards" ? (
+        <div>
         <div className="w-full grid xl:grid-cols-3 gap-3 mb-3 md:grid-cols-2">
           {repositories?.map(({ repository, contributions }) => (
             <RepositoryContributionsCard
-              key={repository.name}
-              repository={repository}
-              contributions={contributions}
+            key={repository.name}
+            repository={repository}
+            contributions={contributions}
             />
-          ))}
+            ))}
+          </div>
         </div>
       ) : format === "text" ? (
         <pre>
           {repositories?.map(({ repository, contributions }) => (
-            <div key={repository.name}>
+            <div key={repository.name} id="repo-content">
               {repository.owner.login}/{repository.name} (
               {contributions.totalCount})
               <br />
               {contributions.nodes.map(({ pullRequest }) => (
-                <span key={pullRequest.id}>
+                <span key={pullRequest.id} id="pr-content">
                   {pullRequest.title} - {pullRequest.state}
                   <br />
                 </span>
               ))}
               <br />
             </div>
           ))}
```

Comment:
You can now replace all of this with generateText to make sure it is the exact same output

```suggestion
          {generateText()}
```

Reply:
![image](https://github.com/Balastrong/github-stats/assets/73685420/5c8558d0-c66a-40f5-90c3-84ddd30c799b)